Dockerfile

```diff
@@ -2,12 +2,12 @@
 # This Dockerfile requires BuildKit to be enabled, by setting the environment variable
 # DOCKER_BUILDKIT=1
 # see https://docs.docker.com/build/buildkit/#getting-started
-ARG BASE_IMAGE="ubuntu:jammy-20230624"
-ARG BASE_DIGEST_AMD64="sha256:b060fffe8e1561c9c3e6dea6db487b900100fc26830b9ea2ec966c151ab4c020"
-ARG BASE_DIGEST_ARM64="sha256:fb4a67ec973b2995214edd101e37a83787b175a16750b372789c8f6314dc20ca"
-ARG JDK_IMAGE="eclipse-temurin:17.0.7_7-jdk-jammy"
-ARG JDK_DIGEST_AMD64="sha256:ab4bbe391a42adc8e590d0c54b3ca7903cbc3b62a3e3b23ac8dce94ebfef6b9e"
-ARG JDK_DIGEST_ARM64="sha256:6af63732bd4bbabe749544e23dac50524b493281208b8b227c7ff87e6d4f2fe9"
+# We use BellSoft's Liberica OpenJDK as base JDK image for two reasons: it provides an Alpine base
+# image with glibc already installed and configured (which is noticeably more performant than musl),
+# and it is slimmer than their JRE offering (which is completely uncompressed)
+ARG BASE_IMAGE="bellsoft/liberica-openjdk-alpine:17.0.7-7"
```

Comment:
actually with patch versioning in the tag, let's remove [these](https://github.com/camunda/zeebe/blob/main/.github/renovate.json#L22-L23) lines in the renovate config to instruct it to update these too

Reply:
I'm still confused on what it updates - the tag, the sha, etc. Do you have a few minutes to explain again? I had understood this config was there so it would only update the sha, but then I guess I would have to remove the minor/patch version from the tag, which goes against the guidelines from @cmur2. So we do want to update minor/patch - and it will also update the SHA pinning?

Reply:
I tested in an example repo [with a minimal config like this](https://confluence.camunda.com/display/HAN/Dockerfile+Base+Images#DockerfileBaseImages-RenovateConfigurationforDockerfile) and given a `FROM user/image:x.y.z@sha256:dddd` it updated the `x.y.z` and  the `dddd` part in conjunction. Not sure about your specific Renovate config but in general it should work.

Reply:
> but then I guess I would have to remove the minor/patch version from the tag, which goes against the guidelines from @cmur2. So we do want to update minor/patch - and it will also update the SHA pinning?

exactly, this is how we initially handled it, a [manually managed major version tag such as `eclipse-temurin:17-jre-focal`](https://github.com/camunda/zeebe/blob/main/Dockerfile#L8) + instructing renovate to only update the digest.

As this is in conflict with the recommendations to actually use a full patch level version tag what this change does we need to change the config now.